cli/main/main.go

```diff
@@ -33,7 +32,7 @@ func main() {
 	app.Author = "Skippbox Kompose Contributors"
 	app.Email = "https://github.com/skippbox/kompose"
 	app.EnableBashCompletion = true
-	app.Before = cliApp.BeforeApp
+	app.Before = command.BeforeApp
 	app.Flags = append(command.CommonFlags())
 	app.Commands = []cli.Command{
 		command.ConvertCommand(),
```

Comment:
I stuck to the way you did it, add the command itself in `BeforeApp` function, so this is not needed I think.